src/amo/containers/App.js

```diff
@@ -73,15 +75,20 @@ export class AppBase extends React.Component {
       InfoDialogComponent,
       MastHeadComponent,
       children,
+      clientApp,
       i18n,
+      lang,
       location,
     } = this.props;
+    const isHomePage = location.pathname === `/${lang}/${clientApp}/`;
```

Comment:
I really wish there was a better way of checking this but react-router isn't too helpful here. It looks like the last `/` is optional, so that should be accounted for.